rust/src/composepost.rs

```diff
@@ -55,6 +58,21 @@ fn postprocess_presets(rootfs_dfd: &openat::Dir) -> Result<()> {
     Ok(())
 }
 
+// We keep hitting issues with the ostree-remount preset not being
+// enabled; let's just do this rather than trying to propagate the
+// preset everywhere.
+fn postprocess_rpm_macro(rootfs_dfd: &openat::Dir) -> Result<()> {
+    let rpm_dir = "usr/etc/rpm";
+    rootfs_dfd.ensure_dir_all(rpm_dir, 0o755)?;
+    let rpm_dfd = rootfs_dfd.sub_dir(rpm_dir)?;
+    rpm_dfd.write_file_with("macros.rpm-ostree", 0o644, |w| -> Result<()> {
+        w.write_all(b"%_dbpath /")?;
+        w.write_all(RPMOSTREE_RPMDB_LOCATION.as_bytes())?;
```

Comment:
While we're here...let's make this `/usr/lib/sysimage/rpm` because that's the new preferred version that is used by e.g. OpenSUSE microos (which is a symlink for now but as the comment says the idea is to switch those in the future).

Reply:
Eh, I guess we can just change this later too at the same time too.

Reply:
Good call, done!